tests/test_starfile.py

```diff
@@ -13,7 +13,7 @@
 
 def normalized_mse(a, b):
     """Return normalized error between two numpy arrays."""
-    return np.sum((a - b) ** 2) ** 0.5 / np.sum(a ** 2) ** 0.5
+    return np.sum((a - b) ** 2) ** 0.5 / np.sum(a**2) ** 0.5
```

Comment:
this is from someone else. I had to use my local black (turn off the precommit hook) to have it change this. then it passed linting. someone (capstone student or harshit) is likely having the same issue as I am.

Reply:
Yes it seems that there has been a black update